File: DocVision.App/app/src/main/java/HospitalDTO.kt
data class HospitalDTO(
    val id: Int,
    val name: String,
    val location: String,
    val locationUrl: String?,
    val openingHours: String?,
    val numberOfAvailableDoctors: Int,
    val rating: Double,
    val img: String?,
    val about: String?,
    val contactNo: String?
)

File: DocVision.Api/DTO/HospitalDTO.cs
namespace FindADoctorApi.DTO
{
    public class HospitalDTO
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Location { get; set; }
        public string LocationUrl { get; set; }
        public string OpeningHours { get; set; }
        public int NumberOfAvailableDoctors { get; set; }
        public double Rating { get; set; }
        public string Img { get; set; }
        public string About { get; set; }
        public string ContactNo { get; set; }
    }
}